airbyte-integrations/connectors/source-amazon-ads/source_amazon_ads/spec.json

```diff
@@ -110,6 +110,16 @@
           "type": "string"
         },
         "uniqueItems": true
+      },
+      "num_workers": {
+        "type": "integer",
+        "title": "Number of concurrent workers",
+        "minimum": 2,
+        "maximum": 10,
+        "default": 10,
+        "examples": [1, 2, 3],
```

Comment:
nit, let's remove `1` from the examples because the minimum is 2 and just in general we've wanted to keep at least 2 threads available at all times